addons/hr_gamification/wizard/gamification_badge_user_wizard.py

```diff
@@ -8,18 +8,14 @@
 class GamificationBadgeUserWizard(models.TransientModel):
     _inherit = 'gamification.badge.user.wizard'
 
-    employee_id = fields.Many2one('hr.employee', string='Employee', required=True)
-    user_id = fields.Many2one('res.users', string='User', related='employee_id.user_id',
-        store=False, readonly=True, compute_sudo=True)
+    employee_id = fields.Many2one('hr.employee', string='Employee', required=False)
+    user_id = fields.Many2one('res.users', string='User', compute='_compute_user_id',
+        store=True, readonly=False, compute_sudo=True)
```

Comment:
This shouldn't be ok on a stable release.

Reply:
The field is stored in gratification .. I though here store=False was doing anything anyways :
:thinking:

Reply:
I just checked again, after installing hr_gamification

![image](https://github.com/odoo/odoo/assets/11148685/7e8a4448-6d13-4086-8a9d-eef085a8b725)

because at 

https://github.com/odoo/odoo/blob/48e40a6c1f449a4093fa2a172bf9b6f3c731ad48/addons/gamification/wizard/grant_badge.py#L7-L12